pio_disassembler.h missing include

After Arch update to GCC 13 `pio_disassembler.h` won't build:

```
/usr/share/pico-sdk/tools/pioasm/pio_disassembler.h:16:25: error: ‘uint16_t’ was not declared in this scope
   16 | std::string disassemble(uint16_t inst, uint sideset_bits, bool sideset_opt);
      |                         ^~~~~~~~
/usr/share/pico-sdk/tools/pioasm/pio_disassembler.h:1:1: note: ‘uint16_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
  +++ |+#include <cstdint>
```

--- a/tools/pioasm/pio_disassembler.cpp
+++ b/tools/pioasm/pio_disassembler.cpp
@@ -7,7 +7,6 @@
 #include <array>
 #include <sstream>
 #include <iomanip>
-#include <cstdint>
 #include "pio_disassembler.h"
 
 extern "C" void disassemble(char *buf, int buf_len, uint16_t inst, uint sideset_bits, bool sideset_opt) {

--- a/tools/pioasm/pio_disassembler.h
+++ b/tools/pioasm/pio_disassembler.h
@@ -10,6 +10,7 @@
 #ifdef __cplusplus
 
 #include <string>
+#include <cstdint>
 
 typedef unsigned int uint;
 

